oca_dependencies.txt

```diff
@@ -0,0 +1 @@
+server-backend
```

Comment:
Runbot fails. Instead of this line, try 
`server-backend https://github.com/OCA/server-backend`

Reply:
Done. Thanks :+1: